hardhat.config.ts

```diff
@@ -1,16 +1,16 @@
-import "dotenv/config";
```

Comment:
Add the file to the eslint in order to check it. Do not change the file based on local eslint settings.

Reply:
It changed the file base on the repo `.prettierrc` config file, I will add it to eslint script